tools/gen_struct_info.py

```diff
@@ -417,7 +417,7 @@ def inspect_code(headers, cpp_opts, structs, defines):
     # Run the compiled program.
     show('Calling generated program...')
     try:
-      info = shared.run_js(js_file[1]).splitlines()
+      info = shared.run_js(js_file[1], shared.NODE_JS).splitlines()
```

Comment:
is this needed? the other change in this PR will ensure `JS_ENGINES` exists with at least node. so it seems like this line would use node anyhow, and we can avoid hard-coding it here?

Reply:
But I want to avoid depending on JS_ENGINES.   Its less error prone if we run these internal tools *always* under node, and never under some other engine.    We should probably have a helper for this and compiler.js and any other internal js tools.. where we want to always use node regardless of JS_ENGINES.

Reply:
But I admit its redundant, I fixed this is two ways.

Reply:
I don't feel strongly about this, but I do think it's unnecessary information to say "node" here. This file doesn't care about node, it just needs *some* JS engine to run in.

Or am I missing something with node's importance here?

Reply:
OK I'll leave this out of this change, but I do want to be clear that outside of testing we only support node.  For compiler.js etc.. and here.